components/filter.css

```diff
@@ -1,15 +1,15 @@
 .left-side-filter {
-  width: 15rem;
+  width: 13rem;
   display: flex;
   flex-direction: column;
   gap: 1.5rem;
   padding: 0.8rem;
   font-family: sans-serif;
 }
-.filter-wrap {
+.filter-wrapperper {
```

Comment:
typo?

Reply:
Thank You So much! That's very much IMP mistake to resolve by me